e2e/playwright/actions/src/tests/library/library-actions.spec.ts

```diff
@@ -0,0 +1,444 @@
+  const requestToJoinMessage = 'Request sent to join this library';
+  const cancelJoinRequestButton = 'Cancel Join Request';
+  const cancelJoinRequestMessage = 'Canceled the request to join the library';
+  const leaveLibraryButton = 'Leave Library';
+  const leaveDialogTitle = 'Leave this library?';
+  const leaveDialogContent = 'Leaving will remove your access.';
+  const leftMessage = 'You have left the library';
+  const favoriteButton = 'Favorite';
+  const removeFavoriteButton = 'Remove Favorite';
+  const deleteButton = 'Delete';
+  const libraryDeletedMessage = 'Library deleted';
+  const siteString = 'site';
+  const loadString = 'load';
+  const domContentLoadedString = 'domcontentloaded';
+
+  const username1 = `user-${Utils.random()}`;
+  const username2 = `user-${Utils.random()}`;
+  const adminLibrary1 = `playwright-A-library-${Utils.random()}`;
+  const adminLibrary2 = `playwright-A-library-${Utils.random()}`;
+  const adminLibrary3 = `playwright-A-library-${Utils.random()}`;
+  const adminLibrary4 = `playwright-A-library-${Utils.random()}`;
+  const adminModerateLibrary1 = `playwright-Am-library-${Utils.random()}`;
+  const adminModerateLibrary2 = `playwright-Am-library-${Utils.random()}`;
+  const adminModerateLibrary3 = `playwright-Am-library-${Utils.random()}`;
+  const adminModerateLibrary4 = `playwright-Am-library-${Utils.random()}`;
+  const user1Library1 = `playwright-U1-library-${Utils.random()}`;
+  const user1Library2 = `playwright-U1-library-${Utils.random()}`;
+  const user1Library3 = `playwright-U1-library-${Utils.random()}`;
+  const user1Library4 = `playwright-U1-library-${Utils.random()}`;
+  const user1Library5 = `playwright-U1-library-${Utils.random()}`;
+  const user2Library1 = `playwright-U2-library-${Utils.random()}`;
+  const user2Library2 = `playwright-U2-library-${Utils.random()}`;
+  const user2Library3 = `playwright-U2-library-${Utils.random()}`;
+  const user2Library4 = `playwright-U2-library-${Utils.random()}`;
+  const user2Library5Delete = `playwright-T-library-${Utils.random()}`;
+  const user2Library6Delete = `playwright-T-library-${Utils.random()}`;
+  const user2Library7Delete = `playwright-T-library-${Utils.random()}`;
+
+  const adminLibraryIds: string[] = [adminLibrary1, adminLibrary2, adminLibrary3, adminLibrary4];
+  const adminModerateLibraryIds: string[] = [adminModerateLibrary1, adminModerateLibrary2, adminModerateLibrary3, adminModerateLibrary4];
+  const user1LibraryIds: string[] = [user1Library1, user1Library2, user1Library3, user1Library4, user1Library5];
+  const siteRoles: string[] = [
+    Site.RoleEnum.SiteManager,
+    Site.RoleEnum.SiteContributor,
+    Site.RoleEnum.SiteCollaborator,
+    Site.RoleEnum.SiteConsumer,
+    Site.RoleEnum.SiteManager
+  ];
+  const user2LibraryIds: string[] = [user2Library1, user2Library2, user2Library3, user2Library4];
+  const user2LibraryIdsDelete: string[] = [user2Library5Delete, user2Library6Delete, user2Library7Delete];
```

Comment:
same here - should be moved to beforeAll as used only there

Reply:
moved in [aa865dc](https://github.com/Alfresco/alfresco-content-app/pull/3545/commits/aa865dcf243348da373c4560117ab99260fb3fe8)